tests/topotests/static_routing_with_ebgp/test_static_routes_topo2_ebgp.py

```diff
@@ -0,0 +1,2338 @@
+                        "next_hop": NEXT_HOP_IP["nh" + str(randnum)][addr_type],
+                        "delete": True,
+                    }
+                ]
+            }
+        }
+        logger.info("Configure static routes")
+        result = create_static_routes(tgen, input_dict_4)
+        assert result is True, "Testcase {} : Failed \n Error: {}".format(
+            tc_name, result
+        )
+
+        step(
+            "After delete of random route , that route only got deleted from"
+            " RIB/FIB other route are showing properly"
+        )
+        nh = NEXT_HOP_IP["nh{}".format(randnum)][addr_type]
+        result = verify_rib(
+            tgen,
+            addr_type,
+            dut,
+            input_dict_4,
+            next_hop=nh,
+            protocol=protocol,
+            expected=False,
+        )
+        assert result is not True, "Testcase {} : Failed \nError: Routes are"
+        " missing in RIB".format(tc_name)
+
+    for addr_type in ADDR_TYPES:
+        input_dict_4 = {
+            "r2": {
+                "static_routes": [
+                    {
+                        "network": PREFIX1[addr_type],
+                        "next_hop": NEXT_HOP_IP["nh" + str(randnum)][addr_type],
+                    }
+                ]
+            }
+        }
+        logger.info("Configure static routes")
+        result = create_static_routes(tgen, input_dict_4)
+        assert result is True, "Testcase {} : Failed \n Error: {}".format(
+            tc_name, result
+        )
+
+    step("Reload the FRR router")
+    # stop/start -> restart FRR router and verify
+    stop_router(tgen, "r2")
+    sleep(5)
+    start_router(tgen, "r2")
+
+    step(
+        "After reload of FRR router , static route "
+        "installed in RIB and FIB properly ."
+    )
+    for addr_type in ADDR_TYPES:
+        # Enable static routes
+        nhp = 1
+        input_dict_4 = {
+            "r2": {
+                "static_routes": [
+                    {
+                        "network": PREFIX1[addr_type],
+                        "next_hop": NEXT_HOP_IP["nh" + str(nhp)][addr_type],
+                    }
+                ]
+            }
+        }
+        logger.info("Verifying %s routes on r2", addr_type)
+        dut = "r2"
+        protocol = "static"
+        result = verify_rib(
+            tgen,
+            addr_type,
+            dut,
+            input_dict_4,
+            next_hop=nh_all[addr_type],
+            protocol=protocol,
+        )
+        assert result is True, "Testcase {} : Failed \nError: Routes are"
+        " missing in RIB".format(tc_name)
+
+    step(
+        "Remove and add the IBGP nbrs using interface shut and"
+        "deactivate/activate from address family"
+    )
+
+    for intfr in range(0, 8):
+        intf = topo["routers"]["r2"]["links"]["r3-link{}".format(intfr)]["interface"]
+        shutdown_bringup_interface(tgen, dut, intf, False)
+
+    sleep(200)
```

Comment:
What are we sleeping for?  Is there a better way to do this?

Reply:
Changes made are:
1. This sleep is as per the test case , sleep timer is optimised to 6 secs.
2. Base config is modified to use bgp aggressive timers.